src/devices/sound/nes_vt_apu.cpp

```diff
@@ -0,0 +1,289 @@
+// license:BSD-3-Clause
+// copyright-holders:David Shah
+/*****************************************************************************
+
+  MAME/MESS VTxx APU CORE
+
+ *****************************************************************************/
+
+#include "nes_vt_apu.h"
+#include "emu.h"
```

Comment:
This won't build properly with precompiled headers disabled - `emu.h` must be the first thing in each compilation unit.